draft-rfcs/working-group-terminology.md

```diff
@@ -2,96 +2,274 @@
+- What is the long-term vision of your group?
+- What do you expect the relationship to the organisation be?
+- How do you want to establish accountability?
+- If applicable, which other working groups or teams do you expect to have close
+  contact with?
+- What are the short-term goals of your group?
+- Who are the initial shepherds/leaders? (This is preferably 2–3 individuals,
+  but not required.)
+- How do you intend to make your work accessible to people outside your group?
+- Where do you see your group needing help?
+- What is your preferred way for the core team to contact you?
+
+### Working Group Evaluation
+
+Working group shepherds should be regularly participating in the project leads
+discussion channels so that the rest of the organisation is aware of the work
+and can help where needed.
+
+If a working group has demonstrated consistent productivity over a significant
+period time, and there is consensus that there is significant future work, it
+may become a Rust team. Conversely if there is consensus that the work is
+"complete" to the point that there's there is little benefit to continuing the
+working group, it may be wound down.
+
+The wind down process of a working group involves communicating the wind down to
+the community and the archival or transfer of ownership of the relevant projects.
+
+## Project Groups
+
+A Project Group is a group of people working on a particular project or
+responsibilities at the behest of an official Rust team. Examples of this would
+include [FFI Unwind], [Inline ASM], and [Safe Transmute].
+
+The goal of a project is build a community or formalise and existing community
+around a particular feature or project in the in the organisation, and use this
+space to discuss and iterate on that feature.
+
+Part of building a community is removing some of the institutional memory that
+develops in the design process, and centralising the information and discussion
+around the feature so that we can provide better visibility into why certain
+decisions and trade offs were made over others.
+
+Previously a lot of the discussion and iteration for large features would
+happen in the initial RFC thread. This leads to a lot of discussion in the top
+of the thread and that has can become completely irrelevant to the current
+iteration.
+
+This process has also been unsuitable to describe features that can take
+multiple years to develop and will become multiple RFCs over the course of its
+design process. Some examples of of this are the "`impl Trait`" and "macros
+2.0" features, where the goals has shifted a lot from the initial RFCs, and it
+can be hard to know their current status.
+
+### Project Group Creation
+
+Since project groups are approved by their relevant parent team over the core
+team, it's up to each team decide their specific requirements. However we
+recommend using the [application checklist](#application-checklist) as the basis
+for process and if needed adding any extra requirements.
```

Comment:
One thing we don't mention here is project group *membership*, right?

I think we want to say that project groups have a notion of members that is maintained at the shepherds' discretion. Typically, people who are productively contributing to the project group for some time will be added as members. But it is not required or expected that all project groups will be large: smaller features may only need one or two people, including the shepherd plus team liasons.